packages/fluttium_runner/test/src/fluttium_runner_test.dart

```diff
@@ -84,6 +85,7 @@ description: test
 - expectNotVisible: "Text"
 - inputText: "Text"
 - takeScreenshot: "Text"
+- wait: "Text"
```

Comment:
This would fail according to the code as it should be an parseable int

Reply:
modified to using plain english durations